### PR TITLE
add exomedepth

### PR DESCRIPTION
**Pull request by gmauro**
Migrated from https://bitbucket.org/crs4/orione-tools

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/exomedepth/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+categories:
+- Sequence Analysis
+description: "ExomeDepth: Calls copy number variants (CNVs) from targeted sequence data"
+homepage_url: https://cran.r-project.org/package=ExomeDepth
+long_description: |
+  Calls copy number variants (CNVs) from targeted sequence data, typically exome sequencing
+  experiments designed to identify the genetic basis of Mendelian disorders.
+name: exomedepth
+owner: crs4
+remote_repository_url: https://github.com/sigu-training/tools/tree/master/tools/exomedepth
```
Comment (bgruening) [inline tools/exomedepth/.shed.yml]: iuc

File: tools/exomedepth/exomedepth.xml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                <validator type="regex" message="Spaces are not allowed">^\S*$</validator>
+            </param>  
+       </repeat>
+       <param name="transition_probability" size="10" type="float" value="0.0001" label="Transition probability" help="Transition probability of the hidden Markov Chain from the normal copy number state to either a deletion or a duplication. The default value (0.0001) expects approximately 20 CNVs genome-wide" />
+    </inputs>
+    <outputs>
+        <data name="output" format="tabular" label="${tool.name} on ${on_string}" />
+    </outputs>
+    <tests>
+      <test>
```
Comment (bgruening) [inline tools/exomedepth/exomedepth.xml]: can you use 4 space intend ...

File: tools/exomedepth/exomedepth.xml [hunk 3]
```diff
@@ -0,0 +1,124 @@
+<tool id="exomedepth" name="ExomeDepth" version="1.1.0">
+    <description>Calls copy number variants (CNVs) from targeted sequence data</description>
+    <requirements>
+        <requirement type="package" version="1.1.10">r-exomedepth</requirement>
```
Comment (bgruening) [inline tools/exomedepth/exomedepth.xml]: 1.1.12 is available ...

Comment (gmauro) [reply tools/exomedepth/exomedepth.xml]: > 1.1.12 is available ...
I know, but the R script doesn't work with that version. We can introduce 1.1.12 in the next wrapper's release

File: tools/exomedepth/.shed.yml (hunk 1 above)
Comment (gmauro) [reply tools/exomedepth/.shed.yml]: updated

File: tools/exomedepth/exomedepth.xml (hunk 2 above)
Comment (gmauro) [reply tools/exomedepth/exomedepth.xml]: updated

**Comment by gmauro** [comment]
we have produced new smaller test-data

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**